lib/govuk-forms-markdown/renderer.rb

```diff
@@ -69,7 +69,7 @@ def triple_emphasis(text)
 
     def link(link, title, content)
       title_attribute = title.nil? ? "" : " title=\"#{title}\""
-      %(<a href="#{link}" class="govuk-link"#{title_attribute} rel="noreferrer noopener" target="_blank">#{content}</a>)
+      %(<a href="#{link}" class="govuk-link"#{title_attribute} rel="noreferrer noopener" target="_blank">#{content} (opens in new tab)</a>)
```

Comment:
At some point we'll need to work out how we're doing translations with this gem, but I think we can put that off for now

Reply:
(this is just me musing, not something you have to do anything about)